.pre-commit-config.yaml

```
@@ -94,3 +94,8 @@ repos:
       - id: taplo-format
         # See options: https://taplo.tamasfe.dev/configuration/formatter-options.html
         args: [--option, "reorder_arrays=true", --option, "reorder_keys=true"]
+
+  - repo: https://github.com/woodruffw/zizmor-pre-commit
+    rev: v0.8.0
```

Should we use latest with this PR or bump in a follow-up?
```suggestion
    rev: v1.0.0
```